pkg/domain/infra/tunnel/containers.go

```diff
@@ -210,3 +212,18 @@ func (ic *ContainerEngine) ContainerCommit(ctx context.Context, nameOrId string,
 	}
 	return &entities.CommitReport{Id: response.ID}, nil
 }
+
+func (ic *ContainerEngine) ContainerExport(ctx context.Context, nameOrId string, options entities.ContainerExportOptions) error {
+	var (
+		err error
+		w   io.Writer
+	)
+	if len(options.Output) > 0 {
+		w, err = os.Create(options.Output)
+		if err != nil {
+			return err
+		}
+
```

Comment:
If you spin another version of this, can you get rid of the extra whitespace here?